www/src/components/account/billing/BillingPricingTable.tsx

```diff
@@ -174,6 +182,10 @@ function BillingPricingTable() {
         <Div body2>Unlimited</Div>
         <Div body2>${clusterMonthlyPricing}/cluster/month</Div>
         <Div body2>${userMonthlyPricing}/user/month</Div>
+        <Div body2>$200/pack of 5 services/month</Div>
```

Comment:
@michaeljguarino hard-coding this for now since I think we'll need the variable for service monthly pricing - TODO fix

Reply:
that's ok yea it's not exposed in the api now (and it's not exactly a line-item due to weirdness w/ usage based billing as well)